### PR TITLE
Add length limitation for description member

### LINKED ISSUE #11961: Issue with "description": Doesn't specify max. length
MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/description

#### What information was incorrect, unhelpful, or incomplete?

Doesn't specify max. length

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/manifest.json/description`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/description
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md
* Last commit: https://github.com/mdn/content/commit/629960fa07e15f2354d75daf14f149d4942539ec
* Document last modified: 2021-10-26T21:28:18.000Z

</details>

### PR DESCRIPTION
**Pull request by AnilSeervi**
#### Summary

- Add length limitation for `description` member
#### Motivation

Open-source
#### Supporting details

https://developer.chrome.com/docs/extensions/mv3/manifest/description/
#### Related issues

Fixes #11961 
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ browser-compat: webextensions.manifest.description
   </tbody>
 </table>
 
-A short description of the extension, intended for display in the browser's user interface.
+A short description(no more than 132 characters) of the extension, intended for display in the browser's user interface.
```
Comment (estelle) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md:33]:
```suggestion
A short description (no more than 132 characters) of the extension, intended for display in the browser's user interface.
```

Comment (AnilSeervi) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md]: Added the space and also removed the `standard-table` class as the styles for that are now directly applied on `<table>`

**Review by zombie — DISMISSED** [review]
I don't believe these limits are the same across browsers.

I would suggest either finding out what are limits in Firefox/Safari/... or more simply just note that browsers might impose different limits for this field.

**Comment by sideshowbarker** [comment]
@AnilSeervi This PR is waiting on your response to https://github.com/mdn/content/pull/12033#pullrequestreview-857201342

**Comment by AnilSeervi** [comment]
> I don't believe these limits are the same across browsers.
> 
> I would suggest either finding out what are limits in Firefox/Safari/... or more simply just note that browsers might impose different limits for this field.

I couldn't find anything on the length of `description` member apart from [chrome docs](https://developer.chrome.com/docs/extensions/mv3/manifest/description/).
let us know if you've come across any such docs.

**Comment by rebloor** [comment]
@AnilSeervi move this forward I would suggest you follow the advice from @zombie and "note that browsers might impose different limits for this field"

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ browser-compat: webextensions.manifest.description
   </tbody>
 </table>
 
-A short description of the extension, intended for display in the browser's user interface.
+A short description (no more than 132 characters) of the extension, intended for display in the browser's user interface.
+
+> Note that browsers might impose different limits for this field.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md:35]: I would suggest rephrasing like this:
```suggestion
A short description of the extension, intended for display in the browser's user interface. In Firefox this value can be up to 132 characters. The limit in other browsers may differ.
```

**Review by rebloor — APPROVED** [review]

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md (hunk 2 above)
Comment (AnilSeervi) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/description/index.md]: Added chrome also according to - https://developer.chrome.com/docs/extensions/mv3/manifest/description/

**Comment by rebloor** [comment]
@AnilSeervi I have approve this. We're waiting on @zombie to review and clear his change requested flag

**Pull request merged**